legwork/source.py

```diff
@@ -125,6 +128,10 @@ def __init__(self, m_1, m_2, ecc, dist, f_orb=None, a=None,
         # reset the arguments with the new converted ones
         m_1, m_2, dist, f_orb, a, ecc = array_args
 
+        # if n_proc not specified, set n_proc=1
+        if n_proc is None:
+            n_proc = 1
+
```

Comment:
Why not just set nproc=1 in the args?

Reply:
you are right! I could!

Reply:
see: ec942f50d4154f3a93ff8531040bed073eda4caf